src/stream/src/executor/hash_join.rs

```diff
@@ -172,6 +173,8 @@ struct JoinSide<K: HashKey, S: StateStore> {
     /// The mapping from input indices of a side to output columes.
     i2o_mapping: Vec<(usize, usize)>,
     i2o_mapping_indexed: MultiMap<usize, usize>,
+    input2inequality_index: Vec<Vec<(usize, bool)>>,
+    state_clean_columns: Vec<(usize, usize)>,
```

Comment:
I have reviewed the code and believe it can work, however, enhancing it with additional comments would aid in comprehending how the code functions.

Reply:
> comprehending

fixed here.